vars/parseStageInfo.groovy

```diff
@@ -0,0 +1,73 @@
+// vars/parseStageName.groovy
```

Comment:
Fixing.   I changed the name when I determined that it was parsing more than just the env.STAGE_NAME for default behavior so that it will work for Matrix stages also.